platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/eventhandling/DefaultTransactionHandlerTests.java

```diff
@@ -119,7 +120,12 @@ void normalOperation(final boolean pcesRound) throws InterruptedException {
                 handlerOutput.reservedSignedState().get().getReservationCount(),
                 "state should be returned with a reservation");
 
-        assertTrue(tester.getSubmittedActions().isEmpty(), "no status should have been submitted");
+        // only the self event reaching consensus should be reported, no freeze action.
+        assertEquals(1, tester.getSubmittedActions().size(), "the freeze status should have been submitted");
```

Comment:
Minor: the assert error message seems inconsistent with the comment at line 123.

Reply:
fixed message.   thanks for the catch.